### PR TITLE
Api 7.3 docs

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
Several documentation updates from API 7.3. See the API changelog as well as https://t.me/bot_api_changes/133

### CODE REVIEW

File: telegram/_bot.py
Comment (harshil21) [inline telegram/_bot.py:1]: `send_location.live_period`, `IQRL.live_period`, and  `InputLocationMessageContent.live_period` docstring also changed - "Period in seconds during which the location will be updated (see <a href="https://telegram.org/blog/live-locations">Live Locations</a>, should be between 60 and 86400, or 0x7FFFFFFF for live locations that can be edited indefinitely."

**Review by harshil21 — CHANGES_REQUESTED** [review]
https://t.me/bot_api_changes/134 can also be added

File: telegram/_bot.py
Comment (Bibo-Joshi) [reply telegram/_bot.py:1]: That has already been partially added in #4247 that's why I didn.t want to do it here. We'll also need a new constant for that

File: telegram/_business.py [hunk 1]
```diff
@@ -246,7 +246,7 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["BusinessIntr
 
 class BusinessLocation(TelegramObject):
     """
-    This object represents the location of a business account.
+    This object Contains information about the location of a Telegram Business account.
```
Comment (aelkheir) [inline telegram/_business.py:249]:
```suggestion
    This object contains information about the location of a Telegram Business account.
```
Notification falsely says "requested your review", so found this :)

**Pull request merged**